frontend/tutor-milton/src/components/Header.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import './Header.css';
+
+const Header = (props) => {
```

Comment:
Instead of passing in props as an object I believe it is currently the convention to separate the props into the specific things you need ({ username, loggedIn }) for example.